bin/list-trello-board-lists.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# Load in our dependencies
+import argparse
+
+from trello import Board, TrelloClient
+
+
+# Define our logic
+def main(api_key, token, board_id):
+    """List out the board lists for our client"""
+    trello_client = TrelloClient(
+        api_key=api_key,
+        token=token,
+    )
+    board = Board(client=trello_client, board_id=board_id)
+    print('Lists')
```

Comment:
No, what I am saying is that because you are not importing `print_function` you have some potential unexpected behavior here. In python2 you are printing a tuple, not using `print` as a function.

Yeah, but `py-trello` is importing `print_function`. https://github.com/sarumont/py-trello/blob/0.4.3/trello/util.py#L2